src/modules/fancyzones/editor/FancyZonesEditor/ViewModels/MonitorViewModel.cs

```diff
@@ -49,13 +51,22 @@ public MonitorViewModel()
             SelectCommand = new RelayCommand<MonitorInfo>(SelectCommandExecute, SelectCommandCanExecute);
 
             Monitors = new ObservableCollection<MonitorInfo>();
+            double maxDimension = 0, minDimension = double.MaxValue;
+
             int i = 1;
             foreach (Screen screen in Screen.AllScreens)
             {
                 Rectangle bounds = screen.Bounds;
+                maxDimension = System.Math.Max(System.Math.Max(maxDimension, bounds.Height), bounds.Width);
+                minDimension = System.Math.Min(System.Math.Min(minDimension, bounds.Height), bounds.Width);
+
                 Monitors.Add(new MonitorInfo(i, screen.DeviceName, bounds.Height, bounds.Width, Settings.CurrentDesktop == i - 1));
                 i++;
             }
+
+            double maxMultipier = 160 / maxDimension;
+            double minMultipier = 96 / minDimension;
```

Comment:
Let's use constants with meaningful names for those numbers.

Reply:
Already done.